packages/website/ts/blockchain.ts

```diff
@@ -2,11 +2,13 @@ import {
     BlockRange,
```

Comment:
@xianny did you build and load the website and exercise the functionality changed? We do not have test coverage for the website so CI won't catch if anything broke. You need to do some manual QA.

Reply:
good catch @fabioberger. I'm getting @fragosti to test the changed components to Portal on staging. I built and ran the other changes locally and they seem fine.